app/controllers/admin/schools_controller.rb

```diff
@@ -9,10 +9,11 @@ class SchoolsController < Admin::BaseController
 
     def index
       @query = params[:query]
+
       @schools = policy_scope(School)
         .includes(:induction_coordinators, :local_authority)
+        .yield_self(&method(:search_query))
```

Comment:
Could this just be `.ransack(induction_coordinators_email_or_urn_or_name_cont: @query).result`?

Reply:
Yes, that should have the same result.

Reply:
OK, I will take a look, I'm not familiar with ransack and didn't twig it could cope with searching relations

Reply:
Yes, seems to work 👍